generators/app/templates/src/app/components/FormInput/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+interface Props {
+  className?: string;
+  disabled?: boolean;
+  error?: string | undefined;
+  errorClassName?: string;
+  inputClassName?: string;
+  isTextarea?: boolean;
+  inputType: string;
+  label?: string | object | undefined;
+  labelClassName?: string;
+  name: string;
+  onBlur?: (e: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>) => void;
+  onChange: (e: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>) => void;
+  onFocus?: (e: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>) => void;
+  placeholder?: string;
+  readOnly?: boolean;
+}
+
+function FormInput({
+  className = '',
+  disabled = false,
+  error = undefined,
+  errorClassName = '',
+  inputClassName = '',
+  isTextarea = false,
+  inputType,
+  label = '',
+  labelClassName = '',
+  name,
+  onBlur,
+  onChange,
+  onFocus,
+  placeholder = '',
+  readOnly = false
+}: Props) {
+  const InputComponent = isTextarea ? 'textarea' : 'input';
+  return (
+    <div className={`column start ${className}`}>
+      {label && (
+        <label htmlFor={name} className={`${labelClassName} m-bottom-1`}>
+          {label}
+        </label>
+      )}
+      <InputComponent
+        className={inputClassName}
```

Comment:
I'd like to add here something like this:

```
className={`${inputClassName} ${styles.input} ${error && touched && submitCount > 0 ? styles.error : ''}}
```
```
.input {
  border-bottom: 1px solid $alto;

  &:focus {
    border-bottom: 2px solid blue;
  }

  &.error {
    border-bottom: 2px solid red;
  }
}
```

We did this for a project, the most important part is this one where we set style to focus and error, we also did some other stuff which I don't think it's useful to have here, but here's the code in case you want to check it out: https://github.com/Wolox/amgen-disease-management-react/tree/development/src/app/components/FormInput

Reply:
We can create a card in Trello for this develop. What do you think? :D @damfinkel

Reply:
Sure